spec/stack.js

```diff
@@ -5,18 +5,56 @@ import Stack from '../src/stack'
 chai.use(chaiChange)
 
 describe('Stack', () => {
-  'use strict'
+  const myStack = new Stack()
 
   it('exists', () => {
     expect(Stack).to.be.a('function')
   })
 
   context('push()', () => {
     it('pushes an element to the top of the stack.', () => {
-      const myStack = new Stack()
+      myStack.push('foo')
+       expect(myStack.container[0]).to.equal('foo')
```

Comment:
I would throw an expect in there that observes the length change to verify that the stack size is increasing.